book.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'item'
```

Comment:
- [ ] [OPTIONAL] As I was suggested by my other fellow coding partner, it is better to collect related files in one folder. So that it will be easier to debug if any error arises in the future and as the site begins to get bigger we do not need to have all the files on the home folder.